app/client/src/components/shared/WaterbodyInfo/index.js

```diff
@@ -1298,65 +1296,57 @@ function UsgsStreamgagesContent({ feature }: { feature: Object }) {
         </tbody>
       </table>
 
-      {sortedPrimaryMeasurements.length === 0 &&
-      secondaryMeasurements.length === 0 ? (
-        <p css={noMeasurementDataStyles}>No data available.</p>
-      ) : (
-        <table css={tableStyles} className="table">
-          <thead>
+      <table css={tableStyles} className="table">
+        <thead>
+          <tr>
+            <th>Category</th>
+            <th>Latest Measurement</th>
+          </tr>
+        </thead>
+        <tbody>
+          {sortedMeasurements.length === 0 ? (
             <tr>
-              <th>Category</th>
-              <th>Latest Measurement</th>
+              <td>
+                <em>No data available.</em>
+              </td>
+              <td>&nbsp;</td>
             </tr>
-          </thead>
-          <tbody>
-            {sortedPrimaryMeasurements}
+          ) : sortedMeasurements.length < 10 ? (
```

Comment:
```suggestion
          ) : sortedMeasurements.length <= 10 ? (
```

This was causing the show more/less button to be visible but not have any data under it. See "HOBBS BK BELOW CAMBRIDGE RES NR KENDALL GREEN, MA", in the example HUC you provided.  This should be a less than equal to check.

Reply:
Thanks, good catch!